spec/features/admin_loads_councillors_spec.rb

```diff
@@ -72,7 +72,6 @@
       visit admin_authority_path(authority)
 
       click_button "Load Councillors"
-
```

Comment:
I think this blank line between the test 'exercise' and 'verify' stages of the test makes it clearer. Would you mind adding it back @hisayohorie ?